### PR TITLE
[#2127]

### PR DESCRIPTION
**Pull request by cadecairos**
1. Add a sanitize function to properly handle text from wikipedia
2. Build text content differently, since wikipedia responses are unpredictable

Lighthouse Ticket: https://webmademovies.lighthouseapp.com/projects/65733/tickets/2127-wikipedia-includes-instead-of-spaces-in-title

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/01e5d66bdc381af/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/01e5d66bdc381af/output.txt

Total script time: 0.91 mins

Server is running on http://pr883.butyr.org/templates/basic/
Run the Test Suite: http://pr883.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0d827208f6bf377/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/0d827208f6bf377/output.txt

Total script time: 2.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/2477bc6aaa95679/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2477bc6aaa95679/output.txt

Total script time: 1.63 mins

Server is running on http://pr883.butyr.org/templates/basic/
Run the Test Suite: http://pr883.butyr.org/test/

File: templates/assets/plugins/wikipedia/popcorn.wikipedia.js [hunk 1]
```diff
@@ -127,7 +150,7 @@
 
         // gets the mobile format, so that we don't load unwanted images when the respose is turned into a documentFragment
         Popcorn.getScript( _href + "api.php?action=parse&prop=text&redirects&page=" +
-          window.escape( _title ) + "&noimages=1&mobileformat=html&format=json&callback=" + _guid );
+           window.escape( _title ) + "&noimages=1&mobileformat=html&format=json&callback=" + _guid );
```
Comment (dseif) [inline templates/assets/plugins/wikipedia/popcorn.wikipedia.js]: too much indentation

**Pull request merged**